lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_writer :name, :letter, :array
```

Comment:
- [ ] **[REQUIRED]** In this case. `name` needs `read` access as well. Similarly, `letter` does not need `write` access as it is only being read and not re-assigned. Because of this issue, the game is no longer running. Please make sure the program runs before you re-submit for review.